backend/src/database/entities/bcws/bcws-personnel-roles.entity.ts

```diff
@@ -28,11 +28,18 @@ export class BcwsSectionsAndRolesEntity {
   })
   expLevel: ExperienceLevel;
 
+  @Column({
+    name: 'rank',
+    type: 'int',
+  })
+  rank: number;
```

Comment:
I think we should make this one nullable since we're probably only collecting 1st/2nd choice rank (and the num of roles a person could have interest/exp is not limited)